scripts/Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM alpine
+FROM alpine AS build
 MAINTAINER chriseth <chris@ethereum.org>
 #Official solidity docker image
 
 #Establish working directory as solidity
 WORKDIR /solidity
+
+# Build dependencies
+ADD /scripts/install_deps.sh /solidity/scripts/install_deps.sh
+RUN ./scripts/install_deps.sh
+
 #Copy working directory on travis to the image
 COPY / $WORKDIR
```

Comment:
By the way (and that might also answer my question above), does this avoid the cache being used as soon as the contents of the copied files change here?

Reply:
Ok, I can also just read the description of this PR :)